agent/util-scripts/pbench-register-tool-set

```diff
@@ -1,119 +1,153 @@
 #!/bin/bash
 # -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
 
-script_path=`dirname $0`
-script_name=`basename $0`
-pbench_bin="`cd ${script_path}/..; /bin/pwd`"
+script_path="$(dirname ${0})"
+script_name="$(basename ${0})"
+pbench_bin="$(realpath -e ${script_path}/..)"
 
 # source the base script
-. "$pbench_bin"/base
+. "${pbench_bin}"/base
 
-# The sole purpose of this script is to ensure a tool
-# of your choosing is used during the execution of a benchmark.
-# For a list of performance tools, look at the
-# $pbench_bin/tool-scripts directory.
+# The sole purpose of this script is to ensure a tool set of your choosing is
+# used during the execution of a benchmark on the optional list of remote
+# hostnames specified. For a list of available tool sets, use the --help
+# option.
 
 # Defaults
 toolset="default"
-remote=""
-label=""
-group=""
+remotes_arg=""
+labels_arg=""
+group_arg=""
+install_arg=""
 default_interval=$(getconf.py interval pbench/tools)
 if [[ -z "${default_interval}" ]] ;then
 	default_interval=3
 fi
 
 function usage() {
-	printf "usage:\n"
-	printf "$script_name --toolset=<tool-set> [--group=<group-name>] [--remote=<remote-host>] [--label=<label>] [--interval=<interval] -- [all tool specific options here]\n"
+	printf -- "usage:\n"
+	printf -- "${script_name} --toolset=<tool-set> [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=<remote-host>[,<remote-host>]] [--labels=<label>[,<label>]]\n"
+	printf -- "\tWhere the list of labels must match the list of remotes.\n"
+	printf -- "\tLike pbench-register-tool, one can specify a single remote\n"
```

Comment:
I'm willing to live with "a single remote" but I thiink it would be better to enumerate the possibilities as I suggested for `pbench-register-tool`: a single remote, a comma-separated list of remotes, or an at sign with a filename.

Reply:
Fixed.